src/artemis/experiment_plans/tests/test_rotation_scan_plan.py

```diff
@@ -358,11 +391,11 @@ def test_rotation_plan_smargon_settings(setup_and_run_rotation_plan_for_tests_st
     assert smargon.x.user_readback.get() == expt_params.x
     assert smargon.y.user_readback.get() == expt_params.y
     assert smargon.z.user_readback.get() == expt_params.z
-    assert omega_vel_set.call_count == 3
-    omega_vel_set.assert_has_calls(
-        [call(DEFAULT_MAX_VELOCITY), call(rotation_speed), call(DEFAULT_MAX_VELOCITY)]
+    assert omega_set.call_count == 6
+    omega_set.assert_has_calls(
+        [call(DEFAULT_MAX_VELOCITY), call(rotation_speed), call(DEFAULT_MAX_VELOCITY)],
+        any_order=True,
```

Comment:
Should: As I put in `conftest` this is an example of where having these as one mock is poor